lib/stdlib/doc/src/lists.xml

```diff
@@ -1126,6 +1126,35 @@ splitwith(Pred, List) ->
 [[a,x,1],[b,y,2],[c,z,3]]</pre>
       </desc>
     </func>
+
+    <func>
+      <name name="uniq" arity="1" since="OTP 25.0"/>
+      <fsummary>Uniques elements of a list preserving ordering.</fsummary>
```

Comment:
I don't think `to unique` is a valid English verb. Maybe
> Removes duplicate elements

? And similarly in a few more places below